config/locales/vi.yml

```diff
@@ -107,13 +107,22 @@ vi:
       rooms_and_suits: "Phòng và những bộ đồ"
       the_best_place_for_relax: "Nơi tốt nhất để thư giãn"
   admins:
+    add_success: "Thêm mới thành công"
+    add_false: "Thêm mới thất bại"
+    update_success: "Cập nhật thành công"
+    update_false: "Cập nhật thất bại"
+    delete_success: "Xóa thành công"
+    delete_false: "Xóa thất bại"
     title: "Trang quản trị"
     home: "Trang chủ"
     dashboard: "Bảng điều khiển"
+    room_not_found: "Phòng không tồn tại"
+    you_sure: "Bạn chắc chứ ?"
     rooms:
-      new: "Thêm mới"
+      submit: Submit
```

Comment:
vi với en như cái nồi lẩu thế

Reply:
> vi với en như cái nồi lẩu thế

nó chơi đồ đấy e, thông cảm :)))))

Reply:
Anh Toàn nói thế mất quan điểm quá anh ơi. Tại trời lạnh nên em cứ thèm nồi lẩu vs vài ly rượu nên nó ms ra như vậy huhuhu